localstack/config.py

```diff
@@ -687,9 +677,7 @@ def legacy_fallback(envar_name: str, default: T) -> T:
     # <ip_address>:<port>(,<ip_address>:port>)*
     GATEWAY_LISTEN,
     # -- Legacy variables
-    EDGE_BIND_HOST,
     EDGE_PORT,
```

Comment:
I guess so. It is used more often in testing than not and `EDGE_PORT` is still widely used in our codebase. It might be better to tackle that post-v3.